composite/apy-finance/src/registry/IAssetAllocation.json

```diff
@@ -0,0 +1,53 @@
+[
```

Comment:
Kind of doesn't make sense to have `IAssetAllocation` abi in a registry folder. Let's just put all ABIs in the `/src/abi` folder.

The second thing is these static ABIs are really hard to identify. What is the origin and source code for these ABIs?  Where can we find some more documentation for these contracts? Please document answers to these questions.